nft-client/src/main.rs

```diff
@@ -561,31 +465,27 @@ async fn main() -> anyhow::Result<()> {
             )
             .context("Could not parse the accounts file.")?;
 
-            let to = match to.0 {
-                types::Address::Account(address) => {
-                    cis2::Receiver::Account(convert_account_address(&address))
-                }
-                types::Address::Contract(address) => {
-                    let receive_name = to_func
-                        .ok_or_else(|| {
-                            anyhow!(
-                                "Transferring to a contract, requires inputting a receive \
-                                 function --to-func"
-                            )
-                        })?
-                        .0;
-                    cis2::Receiver::Contract(convert_contract_address(&address), receive_name)
+            let to = match to {
+                Address::Account(address) => cis2::Receiver::Account(address),
+                Address::Contract(address) => {
+                    let receive_name = to_func.ok_or_else(|| {
+                        anyhow!(
+                            "Transferring to a contract, requires inputting a receive function \
+                             --to-func"
+                        )
+                    })?;
```

Comment:
Could we use `.context` now, the errors should implement `Error`.

Reply:
what error? `to_func` is an option

Reply:
Right, but then we can still use `.context`.